zerver/actions/data_import.py

```diff
@@ -45,17 +52,23 @@ def import_slack_data(event: dict[str, Any]) -> None:
 
             # Try finding the user who imported this realm and make them owner.
             try:  # nocoverage
-                prereg_user = UserProfile.objects.get(
-                    delivery_email__iexact=preregistration_realm.email, realm=realm
+                importing_user = get_user_by_delivery_email(preregistration_realm.email, realm)
```

Comment:
We try to avoid doing `delivery_email__iexact` wherever possible -- better to use the shared helpers as it's harder to mess up.

Reply:
Ah! Thanks for catching and fixing!